tools/ops/policylambda.py

```diff
@@ -28,16 +28,13 @@
 import os
 import string
 import yaml
-import boto3
-from boto3.session import Session
 
 from c7n.config import Config
-from c7n.policy import load as policy_load
-from c7n import mu, resources
+from c7n.loader import PolicyLoader
+from c7n import mu
 
 
-def render(p):
-    policy_lambda = mu.PolicyLambda(p)
+def render_function_properties(p, policy_lambda):
```

Comment:
Being new to the codebase, I really miss docstrings.

Reply:
:-) so its important to keep in mind the scope of this script is degenerate and an example. its for organizations that have arbitrarily decided to not let applications use the api (ie. they've missed the point of the cloud), and have made other arbitrary choices like dictating a single tool for provisioning (terraform, cfn). this script is only provided as a best effort to demonstrate to such organizations how they can enforce their arbitrary limits while using custodian, not as a best practice.